controllers/replica_set_controller.go

```diff
@@ -561,6 +630,14 @@ func buildStatefulSetModificationFunction(mdb mdbv1.MongoDBCommunity) statefulse
 	)
 }
 
+func buildArbitersModificationFunction(mdb mdbv1.MongoDBCommunity) statefulset.Modification {
+	return statefulset.Apply(
+		statefulset.WithReplicas(int(mdb.Spec.Arbiters)),
+		statefulset.WithServiceName(mdb.Name+"-arb-svc"),
```

Comment:
we have a function for the arbiter service name right?

Reply:
thanks for catching this!